### ISSUE TITLE
Possible to add a config option to disable the built-in world save?

### ISSUE THREAD

**Issue by jmattingley23** [espidev/ebackup#50]
I've noticed this plugin triggers it's own world save just before creating a backup, which seems logical but doesn't feel ideal for performance when running something like PaperMC, since this plugin and Paper (or Spigot, Bukkit, etc) will be running competing autosave schedules.

One option could be to just disable the PaperMC autosave and rely on the one from eBackup, but I would probably prefer to use the Paper one since it implements an incremental worldsave strategy where the chunk saving process is spread out over many server ticks to lessen the performance hit, and recommends against saving via any plugins or by manually invoking `/save-all`.  Here's the warning from the spigot logs every time the backup runs:

```
[15:45:00] [Craft Scheduler Thread - 6 - eBackup/INFO]: [eBackup] Starting backup...
[15:45:02] [Server thread/WARN]: A manual (plugin-induced) save has been detected while server is configured to auto-save. This may affect performance.
```

Would it be feasible to add an option to disable the plugin autosave?  I see [here](https://github.com/espidev/ebackup/blob/1a79db0e9535891548da9e7148ff4d2f83046cac/src/main/java/dev/espi/ebackup/BackupUtil.java#L119-L134) you still disable the autosave while the actual compression is taking place, so I'm hoping that removing the preceding save would still be safe.  My thinking is that this shouldn't cause any corruption but would just come at the cost of accepting that the actual world files backed up may sometimes be slightly older than the filename implies since the server autosave would not be perfectly in-step.

Curious to hear your thoughts since you've probably done a lot of testing on this.  Thanks!

**Comment by espidev** [comment]
Yes, as long as the code is able to check and make sure no saves are running at the time of the backup.

EDIT: in regular circumstances, saves would freeze the server and so it likely wouldn't be possible for a backup to happen during a save, i'm not sure if paper acts differently in this case with the feature you describe
